packages/sil-voting-app/src/routes/login/index.tsx

```diff
@@ -39,7 +66,9 @@ const Login = (): JSX.Element => {
         <Typography variant="h6">IOV Voting Dashboard</Typography>
       </Block>
       <Block display="flex">
-        <Button className={classes.button}>LOG IN</Button>
+        <Button className={classes.button} onClick={onLogin}>
+          LOG IN
```

Comment:
I know is kind of big solution for a small thing, but I recommend to use Form here, just because is quite easy to disable the button while the onSubmit callback is being executed.

I know, it is weird because you are going to use a form without any input, but it works fast and in a clean way. Check Login view of bierzo-wallet.

Reply:
Addressed in 1378a3097ebf1cf14e1366d884ada037b92cc7f7.